internal/tracinginterceptor/interceptor.go

```diff
@@ -177,12 +177,55 @@ func (i *Interceptor) CallOneway(ctx context.Context, req *transport.Request, ou
 
 // HandleStream implements interceptor.StreamInbound
 func (i *Interceptor) HandleStream(s *transport.ServerStream, h transport.StreamHandler) error {
-	panic("implement me")
+	parentSpanCtx, _ := i.tracer.Extract(i.propagationFormat, getPropagationCarrier(s.Request().Meta.Headers.Items(), i.transport))
+	span := i.tracer.StartSpan(
+		s.Request().Meta.Procedure,
+		opentracing.ChildOf(parentSpanCtx),
+		ext.SpanKindRPCServer,
+		opentracing.StartTime(time.Now()),
+	)
+	defer span.Finish()
+	err := h.HandleStream(s)
+
+	isApplicationError := err != nil
+	var appErrorMeta *transport.ApplicationErrorMeta
+	if isApplicationError {
+		code := yarpcerrors.FromError(err).Code()
```

Comment:
`yarpcerrors.FromError(err).Code()` is executed twice - one is here, and the other is within `updateSpanWithErrorDetails`.

Maybe we don't need to populate `transport.ApplicationErrorMeta` here.